README.md

```diff
@@ -21,6 +21,10 @@ composer require asdh/save-model
 
 You can publish the config file with:
 ```bash
+php artian save-model:publish
```

Comment:
Thanks for your contribution. Could you please do this change and update the pull request plz.
```bash
php artisan save-model:publish
```

Reply:
Sorry for the typo, I have made the change as well as now make:field command is visible in artisan list